### PR TITLE
Update user model and add cud user actions + tests.

### PR DESCRIPTION
**Pull request by reiterl**
The delete action is not fully working, because of a not implemented structured reverse field.

### CODE REVIEW

**Comment by reiterl** [comment]
Add the payload to user. create, user.update, user.create_temporary, user.update_temporary.
Exclude the decimal field vote_weight, as decimal fields have not been in master, yet.

File: openslides_backend/action/user/create.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from ...models.user import User
```
Comment (normanjaeckel) [inline openslides_backend/action/user/create.py:1]: `from ...models.models import User`

File: openslides_backend/action/user/create_temporary.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from ...models.user import User
```
Comment (normanjaeckel) [inline openslides_backend/action/user/create_temporary.py:1]: `from ...models.models import User`

File: openslides_backend/action/user/delete.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from ...models.user import User
```
Comment (normanjaeckel) [inline openslides_backend/action/user/delete.py:1]: `from ...models.models import User`

File: openslides_backend/action/user/delete_temporary.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+from ...models.user import User
```
Comment (normanjaeckel) [inline openslides_backend/action/user/delete_temporary.py:1]: `from ...models.models import User`

File: openslides_backend/action/user/delete_temporary.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+from ..action import register_action
+from ..default_schema import DefaultSchema
+from ..generics import DeleteAction
+
+
+@register_action("user.delete_temporary")
+class UserDeleteTemporary(DeleteAction):
```
Comment (normanjaeckel) [inline openslides_backend/action/user/delete_temporary.py:8]: @FinnStutzenstein Do we really need this action? It is likely the same as UserDelete?

File: openslides_backend/action/user/update.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
+from ...models.user import User
```
Comment (normanjaeckel) [inline openslides_backend/action/user/update.py:1]: `from ...models.models import User`

File: openslides_backend/action/user/update_temporary.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
+from ...models.user import User
```
Comment (normanjaeckel) [inline openslides_backend/action/user/update_temporary.py:1]: `from ...models.models import User`

File: openslides_backend/action/user/update_temporary.py [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+from ..action import register_action
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+
+
+@register_action("user.update_temporary")
```
Comment (normanjaeckel) [inline openslides_backend/action/user/update_temporary.py:7]: @FinnStutzenstein Do we really need this action?

File: openslides_backend/models/user.py [hunk 9]
```diff
@@ -8,14 +8,76 @@ class User(Model):
     Model for users.
 
     There are the following reverse relation fields:
-        TODO
+    group_$<meeting_id>_ids: (group/user_ids)[];
```
Comment (normanjaeckel) [inline openslides_backend/models/user.py:11]: Please remove this file completly. Models are now in autogenerated models.py

**Review by normanjaeckel — CHANGES_REQUESTED** [review]

**Comment by jsangmeister** [comment]
@reiterl please rebase and fix comments

File: openslides_backend/action/user/create.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/user/create.py:1]: Done.

File: openslides_backend/action/user/create_temporary.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/user/create_temporary.py:1]: Done.

File: openslides_backend/action/user/delete.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/action/user/delete.py:1]: Done.

File: openslides_backend/action/user/delete_temporary.py (hunk 4 above)
Comment (reiterl) [reply openslides_backend/action/user/delete_temporary.py:1]: Done.

File: openslides_backend/action/user/update.py (hunk 6 above)
Comment (reiterl) [reply openslides_backend/action/user/update.py:1]: Done.

File: openslides_backend/action/user/update_temporary.py (hunk 7 above)
Comment (reiterl) [reply openslides_backend/action/user/update_temporary.py:1]: Done.

File: openslides_backend/models/user.py (hunk 9 above)
Comment (reiterl) [reply openslides_backend/models/user.py:11]: Done.

**Comment by reiterl** [comment]
Got problems with NotImplemented Exception at Relations while deleting User. Strange is, I got the errors, even when I try to delete a none existing user.

**Comment by reiterl** [comment]
Norman, can you look at the problems with the delete tests?

**Comment by normanjaeckel** [comment]
Ok, I can do the next days.

**Review by normanjaeckel — APPROVED** [review]

**Comment by normanjaeckel** [comment]
This PR can be rebased.

**Comment by reiterl** [comment]
Rebased and tried to resolve the conflicts. Needed to fix a test.

File: openslides_backend/action/generics.py [hunk 10]
```diff
@@ -227,40 +226,28 @@ def delete_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
...
-                                )
-                            delete_action = delete_action_class(
-                                self.permission, self.database
-                            )
-                            # assume that the delete action uses the standard payload
-                            payload = [{"id": fqid.id}]
-                            self.additional_write_requests.extend(
-                                delete_action.perform(payload, self.user_id)
-                            )
+                    if db_instance.get(field_name):
```
Comment (normanjaeckel) [inline openslides_backend/action/generics.py:229]: I think we have a merge conflict here which @reiterl can not know. I will have a look at it.

**Comment by normanjaeckel** [comment]
Ready to merge.

**Pull request merged**